PoroElastic/PoroElasticity.C

```diff
@@ -568,6 +582,16 @@ bool PoroElasticity::evalSol (Vector& s, const FiniteElement& fe,
   const RealArray& sig = sigma;
   s.insert(s.end(),sig.begin(),sig.end());
 
+  double pressure = press.dot(fe.N);
```

Comment:
isn't it simpler to just pass press here ?

Reply:
Updated. Is this more like what you had in mind?

Reply:
yup.